pkg/obfuscator/domain.go

```diff
@@ -48,6 +48,7 @@ func (d *domainObfuscator) replaceDomains(input string) string {
 				replacement = obfuscatedBaseDomain
 			}
 			output = strings.ReplaceAll(output, m[0], replacement)
+			d.ReplacementTracker.AddReplacement(baseDomain, baseDomain, obfuscatedBaseDomain)
```

Comment:
shouldn't that be:
> d.ReplacementTracker.AddReplacement(baseDomain, m[0], obfuscatedBaseDomain)

?

Reply:
I thought that too, but the unit tests were failing when i tried it.
```
fmt.Println(m)
[docs.okd.io docs. okd.io]
...
expected: map[string]string{"okd.io":"domain0000000002", "openshift.com":"domain0000000001"}
actual  : map[string]string{"docs.okd.io":"domain0000000002", "openshift.com":"domain0000000001"}

```

Reply:
Might it be that the unit tests have different expectations?

Reply:
the map representation now changed a bit, the report will show the right thing. and there is also the side-effect of the generator changes below.

I reckon the tests here then should use the Report directly instead of the map.

Reply:
I changed the unit tests to use the actual reports